rancher-cluster-operations/install-common/output.tf

```diff
@@ -10,3 +10,11 @@ output "rancher_token" {
 output "use_new_bootstrap" {
   value = var.use_new_bootstrap
 }
+
```

Comment:
Are you going to need this in the future?

Reply:
This is leftover from trying out the helm provider's "manifest" experimental feature. Unfortunately it caused a lot of issues at destroy-time, but I'm hoping those get fixed in the future. I'll remove this for now.